modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JaxRSServerCodegen.java

```diff
@@ -166,6 +166,16 @@ public void preprocessSwagger(Swagger swagger) {
         if("/".equals(swagger.getBasePath())) {
             swagger.setBasePath("");
         }
+
+        String host = swagger.getHost();
+        String port = "8080";
+        if(host != null) {
+            String[] parts = host.split(":");
+            if(parts.length > 0) {
```

Comment:
I think it should be `parts.length > 1` instead (no CI failure after changing to `>1`)